invenio_app_rdm/communities_ui/views/ui.py

```diff
@@ -63,6 +68,16 @@ def record_permission_denied_error(error):
     return render_template(current_app.config["THEME_403_TEMPLATE"]), 403
 
 
+def _show_browse_page():
+    """Whether the browse page should be visible in the menu."""
+    community = request.community  # will also be conditioned on collections, etc
+    conf = current_app.config
+    return (
+        conf.get("COMMUNITIES_SHOW_BROWSE_MENU_ENTRY", False)
+        and community["children"]["allow"]
+    )
```

Comment:
nit: IMO the assignments are unnecessary

```suggestion
    return current_app.config.get("COMMUNITIES_SHOW_BROWSE_MENU_ENTRY", False) 
    and request.community["children"]["allow"]
```